### PR TITLE
Develop alexey

### PR DESCRIPTION
**Pull request by balexey88**
Prepare for release `3.4.0`

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
+= 3.4.0 =
+* ENHANCEMENT - remove `udx/lib-settings` package dependency for security reasons. 
+* ENHANCEMENT - remove `udx/lib-utility` package dependency for security reasons.
+* ENHANCEMENT - refactor `Settings` admin page to remove Angular dependency.
+* ENHANCEMENT - include Software Bill of Materials (SBOM) to GitHub release.
+* FIX - update package dependencies for Google Client Library for security reasons.
+* FIX - replace `utf8_encode` with `mb_convert_encoding` to support PHP 8.2 and above [#678](https://github.com/udx/wp-stateless/issues/678).
+* FIX - Fatal Error in `Stateless` mode if GCP access credentials are wrong [#693](https://github.com/udx/wp-stateless/issues/693).
+* COMPATIBILITY - prevent PHP warnings while working with WooCommerce version 8.4.0 and above [696](https://github.com/udx/wp-stateless/issues/696).
+* COMPATIBILITY - avoid conflicts between builtin compatibilities and WP-Stateless Addon plugins.
```
Comment (planv) [inline changelog.txt]: @balexey88 It is already done - should be in the past tense.

File: changes.md [hunk 2]
```diff
@@ -1,3 +1,14 @@
+#### 3.4.0
+* ENHANCEMENT - remove `udx/lib-settings` package dependency for security reasons. 
+* ENHANCEMENT - remove `udx/lib-utility` package dependency for security reasons.
+* ENHANCEMENT - refactor `Settings` admin page to remove Angular dependency.
+* ENHANCEMENT - include Software Bill of Materials (SBOM) to GitHub release.
+* FIX - update package dependencies for Google Client Library for security reasons.
+* FIX - replace `utf8_encode` with `mb_convert_encoding` to support PHP 8.2 and above [#678](https://github.com/udx/wp-stateless/issues/678).
+* FIX - Fatal Error in `Stateless` mode if GCP access credentials are wrong [#693](https://github.com/udx/wp-stateless/issues/693).
+* COMPATIBILITY - prevent PHP warnings while working with WooCommerce version 8.4.0 and above [696](https://github.com/udx/wp-stateless/issues/696).
+* COMPATIBILITY - avoid conflicts between builtin compatibilities and WP-Stateless Addon plugins.
```
Comment (planv) [inline changes.md]: The same https://github.com/udx/wp-stateless/pull/698/files#r1456746534

**Review by planv — APPROVED** [review]

File: changelog.txt (hunk 1 above)
Comment (balexey88) [reply changelog.txt]: Fixed it, thanks!

File: changes.md (hunk 2 above)
Comment (balexey88) [reply changes.md]: Fixed it, thanks!

**Pull request merged**